app/src/main/res/drawable/list_item_background.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape
```

Comment:
The name of this file doesn't really hint at what its use is - what is this used for?

Reply:
Delet this